test/cloudbuild/batch_build.yaml

```diff
@@ -26,9 +26,10 @@ steps:
 options:
   machineType: N1_HIGHCPU_8 # use a fast machine to build because there a lot of work
 images:
-  - "$_GCR_BASE/frontend"
-  - "$_GCR_BASE/scheduledworkflow"
   - "$_GCR_BASE/persistenceagent"
+  - "$_GCR_BASE/scheduledworkflow"
+  - "$_GCR_BASE/frontend"
   - "$_GCR_BASE/viewer-crd-controller"
   - "$_GCR_BASE/visualization-server"
+  - "$_GCR_BASE/inverse-proxy-agent"
```

Comment:
images put here will be pushed to gcr, inverse proxy agent was built but forgot to be pushed